smarts/core/sensors.py

```diff
@@ -191,6 +208,7 @@ class Observation:
     """Amount of simulation time the last step took."""
     step_count: int
     """Number of steps taken by SMARTS thus far."""
+    steps_completed: int
```

Comment:
I am not sure how this observation got in without describing it but:
```suggestion
    steps_completed: int
    """The number of steps this agent has taken within SMARTS."""
```